src/os/bluestore/BlockDevice.h

```diff
@@ -89,7 +89,8 @@ class BlockDevice {
 
   virtual int read(uint64_t off, uint64_t len, bufferlist *pbl,
 	   IOContext *ioc, bool buffered) = 0;
-  virtual int read_buffered(uint64_t off, uint64_t len, char *buf) = 0;
+  virtual int read(uint64_t off, uint64_t len, char *buf,
+           bool buffered) = 0;
```

Comment:
@liewegas thanks for your review. for read_buffered(), it's only used in BlueFS::_read_random(), which is supposed to do byte-granularity length and offset. So I guess it should be fine. Do I need to rename this new "read" function to be "read_random" to avoid confusion?